backends/ebpf/lower.h

```diff
@@ -26,8 +26,12 @@ namespace P4::EBPF {
 /// This pass rewrites expressions which are not supported natively on EBPF.
 class LowerExpressions : public Transform {
     P4::TypeMap *typeMap;
-    /// Cannot shift with a value larger than 5 bits.
-    const int maxShiftWidth = 5;
+    /*
```

Comment:
No "I" in comment. What do you mean with "getting rid of the limt"? You could make this value a std::optional, if it is nullopt there is no shift limit.